cmd/generate.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	pkgcmd "github.com/surajssd/kapp/pkg/cmd"
+)
+
+var (
+	AppFiles []string
```

Comment:
I try to keep variables related to the command (since some are global). May be better to use `GenerateFiles` as the string variable name.